src/compojure/api/routes.clj

```diff
@@ -69,9 +69,14 @@
   (render [_ request]
     (handler request))
 
+  ;; TODO Implement compojure.response/Sendable for async support
```

Comment:
The problem is that I don't understand when it would be called. When does `render` get called for a Route? Sendable is the async version of Renderable.